lib/src/styles/theme/themes.dart

```diff
@@ -44,6 +44,7 @@ class _ArDriveAppState extends State<ArDriveApp> {
   void initState() {
     super.initState();
 
+    // ignore: deprecated_member_use
     var window = WidgetsBinding.instance.window;
```

Comment:
I know there are more ways of accessing this object, we'll have to have a tech_debt

https://docs.flutter.dev/release/breaking-changes/window-singleton